src/samplers/tcp/stat.rs

```diff
@@ -232,6 +238,22 @@ impl TcpStatistic {
             binary_path: None,
             sub_system: None,
         };
+        let tcp_duplicate_probe = Probe {
+            name: "tcp_validate_incoming".to_string(),
+            handler: "trace_validate_incoming".to_string(),
+            probe_type: ProbeType::Kernel,
+            probe_location: ProbeLocation::Entry,
+            binary_path: None,
+            sub_system: None,
+        };
+        let tcp_ooo_probe = Probe {
```

Comment:
this probe is identical as the above one. We can just introduce a probe called

tcp_validate_incoming_probe and attach it to both telemetries

Self::Duplicate => vec![tcp_validate_incoming_probe],
Self::OutOfOrder => vec![tcp_validate_incoming_probe],

Reply:
Cool, I was wondering if I could do that :)